test/run-drun/stable-mem-grow.mo

```diff
@@ -0,0 +1,55 @@
+import P "mo:⛔";
+import StableMemory "stable-mem/StableMemory";
+
+actor {
+
+  stable var n : Nat32 = 0;
+
+  system func preupgrade() {
+    P.debugPrint("upgrading from " # debug_show n);
+    let m = StableMemory.grow(1);
+
+    // check all pages clear
+    var i : Nat32 = 0;
+    let max = StableMemory.size() * 65536;
+    while (i < max) {
+      assert (StableMemory.loadNat32(i) == 0);
+      i += 4
+    };
+
+    n += 1;
+
+  };
+
+  public func testGrow() : async () {
+    var i : Nat32 = 0;
+    while (i < 10) {
```

Comment:
Don't you like for loops? :)

Reply:
No base in tests, so I'd need to implement an iterator first. So, no.